includes/displayBikes.php

```diff
@@ -0,0 +1,48 @@
+<?php
+if(!defined('SAFETORUN')){
+    echo 'You can\'t run this file on its own.';
+    die;
+}
+
+/**
+ * take in a db connection and return card html objects with the information from the bikes table in them
+ *
+ * @param PDO $aDB - a db connection
+ * @return void
+ */
+function printCards (PDO $aDB){
```

Comment:
This could be unit tested if split into two parts (one to get the data from the db, and the other to display). Just a suggestion!